inc/ticket.class.php

```diff
@@ -2766,8 +2774,6 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
                         foreach ($tomerge as $key => $fup2) {
                            $fup2['items_id'] = $input['_mergeticket'];
                            $fup2['sourceitems_id'] = $item->getID();
```

Comment:
```suggestion
                           $fup2['sourceitems_id'] = $item->getID();
                           $fup2['content'] = $DB->escape($fup2['content']);
```
This will prevent failure if contents contains a `'`. Maybe it was the problem on #5338.